apps/programmes/urls.py

```diff
@@ -8,4 +8,5 @@
 
 urlpatterns = [
     path("", include(router.urls)),
+    path('programmes/<uuid:uuid>/courses/', ProgrammeViewSet, name='programme-courses')
```

Comment:
this is not necessary, it's included in the router of line 7

Reply:
okay I will remove this